.github/workflows/test.yaml

```diff
@@ -0,0 +1,19 @@
+on: [push]
+
+jobs:
+  spell-check:
+    runs-on: ubuntu-latest
+    name: Test the spell check action
+    steps:
+      # To use this repository's private action,
+      # you must check out the repository
+      - name: Checkout
+        uses: actions/checkout@v4
+      - name: Spell check action
+        uses: ./ # Uses an action in the root directory
```

Comment:
You have this as a TBD in your opening PR comment, but I don't think it's TBD _here_ - it's TBD for porting this action into other repos

Reply:
Either way. I could implement it all here, or in a next PR.

Reply:
I'd probably rather have this all merged in so we can test with a built image.

Reply:
> I'd probably rather have this all merged in so we can test with a built image.

So, implement it all here? that's fine with me

Reply:
Sorry, I meant merge in the current state. That will trigger a build and push of the image, so we can then test with a pulled image in the next update.

Reply:
It needs to be this version of the Docker image/script, with the changes to inputs and outputs.